hr_attendance_reason/models/hr_attendance_reason.py

```diff
@@ -8,9 +8,11 @@
 class HrAttendanceReason(models.Model):
     _name = "hr.attendance.reason"
     _description = "Attendance Reason"
+    _order = "sequence"
```

Comment:
Remember always to put id as fallback.

```suggestion
    _order = "sequence,id"
```